<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibraryReadingIconGridViewer</class>
 <widget class="QWidget" name="LibraryReadingIconGridViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>799</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>849</width>
    <height>799</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="LibraryReadingIconGridViewerCont" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>849</width>
       <height>799</height>
      </size>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="LongPressWidget" name="mainBookInfoCont" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>265</height>
         </size>
        </property>
        <widget class="QFrame" name="currentBookCont">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>849</width>
           <height>375</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="currentBookBg" native="true">
            <layout class="QGridLayout" name="gridLayout">
             <property name="margin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="2">
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>28</width>
                 <height>57</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="currentBookTopLeft">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="currentBookTopCenter">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>7</width>
                 <height>57</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="currentBookBottoRight">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>28</width>
                 <height>57</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="currentBookTopRight">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="currentBookBottomCenter">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="currentBookBottomLeft">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="currentBookCenterLeft">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="currentBookCenterRight">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <spacer name="verticalSpacer_8">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>28</width>
                 <height>57</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="2">
              <spacer name="verticalSpacer_9">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>28</width>
                 <height>57</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>7</width>
                 <height>57</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="1">
              <widget class="QWidget" name="currentBookCoverCont" native="true">
               <widget class="QLabel" name="currentBookCoverLbl">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>168</width>
                  <height>253</height>
                 </rect>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
               <widget class="QLabel" name="newTagLbl">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>198</width>
                  <height>308</height>
                 </rect>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>140</width>
                  <height>218</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>198</width>
                  <height>308</height>
                 </size>
                </property>
                <property name="text">
                 <string>Nuevo</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignBottom|Qt::AlignHCenter</set>
                </property>
                <property name="margin">
                 <number>13</number>
                </property>
                <property name="indent">
                 <number>0</number>
                </property>
               </widget>
               <widget class="DegreesLabel45" name="sampleTagLbl">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>99</width>
                  <height>99</height>
                 </rect>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>70</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>99</width>
                  <height>99</height>
                 </size>
                </property>
                <property name="text">
                 <string>Muestra</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                </property>
                <property name="indent">
                 <number>0</number>
                </property>
               </widget>
               <widget class="DegreesLabel45" name="subscriptionTagLbl">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>99</width>
                  <height>99</height>
                 </rect>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>70</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>99</width>
                  <height>99</height>
                 </size>
                </property>
                <property name="text">
                 <string>Subscripción</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                </property>
                <property name="indent">
                 <number>0</number>
                </property>
               </widget>
               <widget class="QLabel" name="coverAuthorLbl">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>253</width>
                  <height>314</height>
                 </rect>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignHCenter|Qt::AlignTop</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="indent">
                 <number>0</number>
                </property>
               </widget>
               <widget class="QLabel" name="coverTitleLbl">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>253</width>
                  <height>281</height>
                 </rect>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
               <zorder>currentBookCoverLbl</zorder>
               <zorder>coverAuthorLbl</zorder>
               <zorder>coverTitleLbl</zorder>
               <zorder>newTagLbl</zorder>
               <zorder>sampleTagLbl</zorder>
               <zorder>subscriptionTagLbl</zorder>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="currentBookData" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="currentBookData2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="readingLbl">
               <property name="text">
                <string>Leyendo...</string>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ElidedLabel" name="bookTitleLbl">
               <property name="text">
                <string>Las horas oscuras</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="ElidedLabel" name="authorLbl">
               <property name="text">
                <string>Juan Francisco Ferrándiz</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="indent">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="pagCont">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="readingPercentLbl">
                 <property name="text">
                  <string>40</string>
                 </property>
                 <property name="indent">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="percentLbl">
                 <property name="text">
                  <string>%</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="readLbl">
                 <property name="text">
                  <string>Leído</string>
                 </property>
                 <property name="indent">
                  <number>0</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>57</width>
                   <height>28</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="lastAccessCont">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="lastAccessLbl">
                 <property name="text">
                  <string>Último acceso:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="dateAccessLbl">
                 <property name="text">
                  <string>Fecha de acceso</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>57</width>
                   <height>28</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>28</width>
                 <height>57</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>57</width>
              <height>28</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="iconGridViewerCont" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>195</horstretch>
          <verstretch>27</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>14142</width>
          <height>14142</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QFrame" name="book4Cont">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="LibraryIconGridViewerItem" name="book4" native="true">
              <property name="minimumSize">
               <size>
                <width>190</width>
                <height>240</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>269</width>
                <height>141421</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>14</width>
                <height>28</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QFrame" name="book5Cont">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="LibraryIconGridViewerItem" name="book5" native="true">
              <property name="minimumSize">
               <size>
                <width>190</width>
                <height>240</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>269</width>
                <height>14142</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>14</width>
                <height>28</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QFrame" name="book6Cont">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="LibraryIconGridViewerItem" name="book6" native="true">
              <property name="minimumSize">
               <size>
                <width>190</width>
                <height>240</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>269</width>
                <height>14142</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>14</width>
                <height>28</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="LibraryPageHandler" name="VerticalPagerHandler" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>47</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>71</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ElidedLabel</class>
   <extends>QLabel</extends>
   <header>ElidedLabel.h</header>
  </customwidget>
  <customwidget>
   <class>DegreesLabel45</class>
   <extends>QLabel</extends>
   <header>DegreesLabel45.h</header>
  </customwidget>
  <customwidget>
   <class>LibraryPageHandler</class>
   <extends>QWidget</extends>
   <header>LibraryPageHandler.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LibraryIconGridViewerItem</class>
   <extends>QWidget</extends>
   <header location="global">LibraryIconGridViewerItem.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LongPressWidget</class>
   <extends>QWidget</extends>
   <header>LongPressWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
